scripts/update_teams.py

```diff
@@ -5,13 +5,15 @@
 #  - python
```

Comment:
Ah. @pelson is not using the best feature of his own tool. That should be `python=2.7`.

(@ericdill solution is also desirable though.)